LICENSE.md

```diff
@@ -23,14 +22,19 @@ SPDX-License-Identifier: Apache-2.0
 
 This enables machine processing of license information based on the SPDX License Identifiers that are here available: http://spdx.org/licenses/
 
+## Third Party Licenses
 
-# Third Party Licenses
+All files listed in the path `test/packs/ARM/RteTest_DFP/0.1.1/Device` are licensed under
+[Apache License 2.0](http://www.apache.org/licenses/LICENSE-2.0) and are attributed to ARM Limited. The files originate
+from https://github.com/ARM-software/CMSIS_5/tree/develop/Device/ARM.
 
-All files listed in the path `test/packs/ARM/RteTest_DFP/0.1.1/Device` are licensed under [Apache License 2.0](http://www.apache.org/licenses/LICENSE-2.0) and are attributed to ARM Limited. The files originate from https://github.com/ARM-software/CMSIS_5/tree/develop/Device/ARM.
+## External Dependencies
 
-# External Dependencies
+The components listed below are not redistributed with the project but are used internally for building, development,
+or testing purposes.
 
-The components listed below are not redistributed with the project but are used internally for building, development, or testing purposes.
+<!-- markdownlint-capture -->
```

Comment:
This looks tedious to me. How do people know where they would need to place certain "exceptions" like disabling MD013.

Reply:
This can be read in the markdown-lint documentation at https://github.com/DavidAnson/markdownlint

Where would be a good place to add the link to?

Reply:
Maybe as part of the output in the linter check github action?
We should also consider extending the repository README to explain "contributing". As we have a number of checks that need to pass before we are going to accept/merge a PR.